platform/podjs/bin/genPodjs.js

```diff
@@ -0,0 +1,75 @@
+const MagicString = require("magic-string");
```

Comment:
Since this seems to be the only dependency it's got, and apparently the changes to package-lock.json seem to imply the inclusion of the new `podjs` is bringing over *lots* of dependencies to the feature modules, I wonder if this wouldn't be better on its own submodule, with just the dependencies needed for it, and of course tests.